examples/3d/render_to_texture.rs

```diff
@@ -61,30 +62,44 @@ fn main() {
 }
 
 // Add 3D render phases for FIRST_PASS_CAMERA.
-fn extract_first_pass_camera_phases(mut commands: Commands, active_cameras: Res<ActiveCameras>) {
-    if let Some(camera) = active_cameras.get(FIRST_PASS_CAMERA) {
-        if let Some(entity) = camera.entity {
-            commands.get_or_spawn(entity).insert_bundle((
-                RenderPhase::<Opaque3d>::default(),
-                RenderPhase::<AlphaMask3d>::default(),
-                RenderPhase::<Transparent3d>::default(),
-            ));
-        }
+fn extract_first_pass_camera_phases(
+    mut commands: Commands,
+    query: Query<Entity, With<FirstPassCamera>>,
+) {
+    for entity in query.iter() {
+        commands.get_or_spawn(entity).insert_bundle((
```

Comment:
I understand there's likely only a handful of these cameras, but is it possible to switch this to use the insert_or_spawn_batch pattern for extraction used in https://github.com/bevyengine/bevy/pull/4179? This could speed notably speed up extraction here.

Reply:
Hmm wait this should actually just extract the active camera. Might as well make this O(1)